src/compiler/types.ts

```diff
@@ -284,6 +284,7 @@ namespace ts {
         IndexedAccessType,
         MappedType,
         LiteralType,
+        ImportTypeNode,

```

Comment:
@weswigham is in intentional that this is the only `SyntaxKind` with the suffix Node?

Reply:
T.T no, I just unconsciously reused the interface name (which always uses `Node`) for the syntax kind. Will fix.